test/fixtures/render/enum-default-multiple.json

```diff
@@ -199,7 +199,7 @@
                                   "content": "application/json"
                                 }
                               },
-                              "content": "{\n  \"enumField\": \"g\"\n}"
+                              "content": "{\n  \"enumField\": \"e\"\n}"
```

Comment:
Wouldn't this be a regression? Previously the last default value took precedence and now the first one does.

```apib
# Group Example
# GET /

+ Response 200 (application/json)
    + Attributes
        + enumField (enum)
            + Default: e
            + Default: f
            + Default: g
```

Although there is no warning saying I am shadowing the previous default values, but we are generally preferring the last one before and in other code paths so I think we should continue to do so for consistency instead of changing the behaviour during refactoring.